synapse/events/__init__.py

```diff
@@ -146,7 +146,7 @@ def __contains__(self, field):
         return field in self._event_dict
 
     def items(self):
-        return self._event_dict.items()
+        return list(self._event_dict.items())
```

Comment:
Wouldn't it be nicer to do `list(iteritems(...))`?

Reply:
That's about twice as slow on Python 2, unfortunately:

```
~ $ python3 -m timeit "list(globals().items())"
1000000 loops, best of 3: 0.75 usec per loop
~ $ python -m timeit "list(globals().items())"
1000000 loops, best of 3: 0.67 usec per loop
~ $ python -m timeit "list(globals().iteritems())"
1000000 loops, best of 3: 1.02 usec per loop
```

Reply:
Well that's fun :/